runtime/account_test.go

```diff
@@ -1069,84 +1065,6 @@ func TestRuntimePublicKey(t *testing.T) {
 		assert.Contains(t, err.Error(), "value of type `PublicKey` has no member `validate`")
 	})
 
-	t.Run("IsValid", func(t *testing.T) {
```

Comment:
Maybe instead of removing the tests, keep them and inject `validatePublicKey` once with it returning true and assert the program executes, and add another time with `validatePublicKey` returning false and assert the program aborts